### PR TITLE
Add 1.3.0 release notes.

### PR DESCRIPTION
**Pull request by arthurevans**
Staged here: https://1-3-0-dot-polymer-project.appspot.com/1.0/docs/release-notes.html#v1-3-0

### CODE REVIEW

File: 1.0/docs/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,295 @@ subtitle: About this release
...
+
+
+-   [#1727](https://github.com/Polymer/polymer/issues/1727). Eliminate the need
+    to add `:host` or other selector in front of `::content` in some situations.
+    This applies if the `<content>` tag is a top-level local DOM child, and there
+    are no other local DOM nodes that could match the selector, for example:
+
+        <template>
+          <content></content>
+        </template>
```
Comment (azakus) [inline 1.0/docs/release-notes.md]: Maybe add an example where you would still need to use a wrapper around `<content>`?

**Comment by azakus** [comment]
LGTM

**Comment by robdodson** [comment]
This URL: https://1-3-0-dot-polymer-project.appspot.com/1.0/docs/release-notes.html#1.3.0-dynamic-fns is missing a v. Should be https://1-3-0-dot-polymer-project.appspot.com/1.0/docs/release-notes.html#v1-3-0-dynamic-fns

**Comment by arthurevans** [comment]
Added example, fixed 3 bad links.

**Comment by bicknellr** [comment]
GitHub seems to have changed their URL scheme. 'pulls' needs to be 'pull'

**Comment by arthurevans** [comment]
Thanks @bicknellr ... I think we already have a community PR to fix those links. (I shoudn't have tried to get fancy there with the links... I think they would've worked OK if I'd left them all as issue links.)

**Comment by bicknellr** [comment]
Oh, ok! (IMHO the real URLs are preferable so they can work everywhere.)

**Pull request merged**